numpy/_core/tests/test_defchararray.py

```diff
@@ -270,103 +273,119 @@ def test_find(self, dtype, encode):
                            [[1, -1], [-1, -1], [-1, -1]])
         assert_array_equal(A.find([encode('1'), encode('P')]),
                            [[-1, -1], [0, -1], [0, 1]])
-        C = self.C.astype(dtype)
+        C = (np.array(['ABCDEFGHIJKLMNOPQRSTUVWXYZ',
+                            '01234567890123456789012345'])
```

Comment:
Let's make this line up too

```suggestion
        C = (np.array(['ABCDEFGHIJKLMNOPQRSTUVWXYZ',
                       '01234567890123456789012345'])
```